setup-rhmap.sh

```diff
@@ -28,7 +28,8 @@ printf "\rProgress : [${done// /#}${undone// /-}] ${precentage}%%"
 # login to docker
 docker login
 
-# Start minishift and find address
+# Create profile start minishift and find address
+minishift profile set rhmap-4x
```

Comment:
Looks good - do you think there's any value in allowing the user the option to specify the minishift profile name also?

Reply:
removing manual steps